vllm/model_executor/layers/quantization/utils/w8a8_utils.py

```diff
@@ -10,7 +10,7 @@
 # providing scaling factor for result. This value is created
 # as global value to avoid multiple tensor allocations, and
 # can be removed once pytorch fixes the bug.
-TORCH_SCALED_MM_SCALE_RESULT = torch.ones(1).cuda() if is_hip() else None
+TORCH_DEVICE_IDENTITY = torch.ones(1).cuda() if is_hip() else None
```

Comment:
Same issue in initializing a global CUDA tensor as in gradlib `GemmTuner.py`, with a similar workaround.

I'm concerned that this is a premature optimization. Even if this was done as a multiple allocation: this would not be an issue in CUDA graph mode, while the overhead for eager mode remains to be determined (for such a small tensor, PyTorch's allocator should be able to supply a cached allocation).

Reply:
Ah I see this is a mistake by a [PR in upstream](https://github.com/vllm-project/vllm/pull/7210).

I'll agree that this one shouldn't be changed in this PR. However, I would suggest not compounding the error in `gradlib`.

As for numbers: if this was initialized as `None` as a global and only initialized as a Tensor once when first used, which is what I suggested, you would not have any performance concerns on top of what's done currently.

Also, I shouldn't have to explain this to an experienced engineer, but conversations should be marked resolved by the conversation starter when possible, not when the PR author wishes to close discussions they feel are inconvenient.